package.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dependencies": {},
+  "devDependencies": {
+    "prettier": "https://github.com/prettier/prettier/tarball/master"
```

Comment:
I have to use the current master version, since [v1.9](https://github.com/prettier/prettier/milestone/7) has not been released. It adds the option to not reformat prose into 80-column text, which messes up git diffs.